pkg/model/components/etcdmanager/options.go

```diff
@@ -65,34 +69,53 @@ func (b *EtcdManagerOptionsBuilder) BuildOptions(o interface{}) error {
 	return nil
 }
 
-var etcdSupportedImages = map[string]string{
-	"3.2.24": "registry.k8s.io/etcd:3.2.24-1",
-	"3.3.10": "registry.k8s.io/etcd:3.3.10-0",
-	"3.3.17": "registry.k8s.io/etcd:3.3.17-0",
-	"3.4.3":  "registry.k8s.io/etcd:3.4.3-0",
-	"3.4.13": "registry.k8s.io/etcd:3.4.13-0",
-	"3.5.0":  "registry.k8s.io/etcd:3.5.0-0",
-	"3.5.1":  "registry.k8s.io/etcd:3.5.1-0",
-	"3.5.3":  "registry.k8s.io/etcd:3.5.3-0",
-	"3.5.4":  "registry.k8s.io/etcd:3.5.4-0",
-	"3.5.6":  "registry.k8s.io/etcd:3.5.6-0",
-	"3.5.7":  "registry.k8s.io/etcd:3.5.7-0",
-	"3.5.9":  "registry.k8s.io/etcd:3.5.9-0",
+// etcdVersion describes how we want to support each etcd version.
+type etcdVersion struct {
+	Version          string
+	Image            string
+	SymlinkToVersion string
 }
 
-func etcdSupportedVersions() []string {
-	var versions []string
-	for etcdVersion := range etcdSupportedImages {
-		versions = append(versions, etcdVersion)
-	}
-	sort.Strings(versions)
+var etcdSupportedImages = []etcdVersion{
+	{Version: "3.4.0", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.1", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.2", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.3", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.4", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.5", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.6", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.7", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.8", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.9", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.10", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.11", SymlinkToVersion: "3.4.13"},
+	{Version: "3.4.12", SymlinkToVersion: "3.4.13"},
```

Comment:
We only supported 3.4.3 and 3.4.13, we should not add more here.
Similar to the 3.5.x, we should skip 2, 5 and 9.